### PR TITLE
doc(changelog): mention CVE fixes

### PR DESCRIPTION
**Pull request by weihanglo**
### What does this PR try to resolve?

CVE fixes definitely worth a place in changelog.

### How should we test and review this PR?

I am not sure about where and how we should place them. 

- In `Fixed` section with a ⚠️ emoji?
- Got their own section such like this PR does?
- Should we use links to GitHub advisories or other site?
- Should we link to official blogpost or the one from official security advisory, which got a PGP key attached.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,15 @@
 ## Cargo 1.64 (2022-09-22)
 [a5e08c47...rust-1.64.0](https://github.com/rust-lang/cargo/compare/a5e08c47...rust-1.64.0)
 
+### ⚠️ Fixes of security vulnerabilities
```
Comment (hi-rustin) [inline CHANGELOG.md:22]: Maybe this should be part of `Fixed`?

**Comment by ehuss** [comment]
I would probably just put it in the "Fixed" section. I don't feel like these are severe enough to warrant special attention. 

I also probably wouldn't bother with the link to the blog post. I think the CVE link more or less has all the same content.

I would also probably render it a little differently to match the rendering of the other entries. Instead of linking the entire description, I would just link the `CVE` code, and also include the pull request. Maybe something like:

- [CVE-2022-36113](https://github.com/rust-lang/cargo/security/advisories/GHSA-rfj2-q3h3-hm5j): Extracting malicious crates can corrupt arbitrary files [#11089](https://github.com/rust-lang/cargo/pull/11089) [#11088](https://github.com/rust-lang/cargo/pull/11088)

**Comment by weihanglo** [comment]
Updated as @ehuss suggested. Thanks!

**Comment by ehuss** [comment]
Thanks!

@bors r+

**Pull request merged**